samples/Java/Basic/ReceiveUsingSequenceNumber/README.md

```diff
@@ -0,0 +1,30 @@
+# Receive events from Azure Event Hubs using Java
+
+This sample shows how to receive events from a particular Event Hub partition based on a sequence number. This is a lower-level API gesture that most applications will not use, but rather lean on the Event Processor Host to manage partition ownership and check-pointing. The [Event Processor Sample](../EventProcessorSample) shows this higher level functionality.
```

Comment:
I would love to see if we can elaborate "lower-level api gesture".
Briefly explain - that since, each eventhub have multiple partitions - and since this receiver can only receive off of a single partition - we also provide a library - which developers can use to receive off of all partitions.

Reply:
Ack, will update the changes. Thank you :)